app/src/androidTest/java/com/github/sdpcoachme/schedule/ScheduleActivityTest.kt

```diff
@@ -63,13 +69,13 @@ class ScheduleActivityTest {
             end = currentMonday.plusDays(3).atTime(15, 0, 0).toString(),
             description = "Learn about the latest design improvements to help you build personal dynamic experiences with Material Design.",
         ),
-        Event(
+        /*Event(
```

Comment:
Is this commented out intentionally? If yes, I would recommend to delete it to increase readability